config/candidate-api.yml

```diff
@@ -37,6 +37,8 @@ paths:
           "$ref": "#/components/responses/Unauthorized"
         '422':
           "$ref": "#/components/responses/ParameterMissing"
+        '422':
```

Comment:
This raises `duplicated mapping key` in the swagger editor.  Maybe `oneOf`?https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/